MODULE.bazel

```diff
@@ -8,7 +8,7 @@ module(
 
 # Lower-bounds for runtime dependencies.
 # Do not bump these unless rules_js requires a newer version to function.
-bazel_dep(name = "aspect_bazel_lib", version = "1.39.0")
+bazel_dep(name = "aspect_bazel_lib", version = "1.40.0")
```

Comment:
thanks for resisting the urge to jump to latest 😉

Reply:
1.40.0 is the latest 1.x branch anyway 😉